src/types/types.tsx

```diff
@@ -88,3 +88,26 @@ export interface Update {
   summary: string;
   lawFirm: string;
 }
+
+export enum GreenStatusOptions {
+  'In Progress',
+  'New Case',
+  'Settled',
+  'Appeal',
+  'Payment Processing',
+  'Payment Distributed',
+}
+
+export enum YellowStatusOptions {
+  'Pending',
+}
+
+export enum RedStatusOptions {
+  'Action Required',
+}
+
+export const StatusOptions: string[] = [
+  ...(Object.values(GreenStatusOptions) as string[]),
+  ...(Object.values(YellowStatusOptions) as string[]),
+  ...(Object.values(RedStatusOptions) as string[]),
```

Comment:
For the sake of simplifying this (since I honestly don't know what's happening) let's assume that for any status input that we don't recognize, we'll return a default grey color. That way we can avoid introducing new errors into the codebase.

So, let's get rid of this `StatusOptions` type and change the util function accordingly.

Reply:
I removed the `StatusOptions` type and deleted the extra code in at the top of the function that dealt with the error.  Let me know if that isn't what you meant and I can change.